src/core/a-assets.js

```diff
@@ -83,9 +83,11 @@ registerElement('a-asset-item', {
     attachedCallback: {
       value: function () {
         var self = this;
-        var src = this.getAttribute('src');
+        var src;
 
+        src = this.getAttribute('src');
```

Comment:
If we're going to do separate declarations, I prefer to keep them pure declarations when we can. If the intent is to separate declaration and logic and make the style 100% unquestionable with no exception, this does that.

I've found mixing declarations and initializations is what requires me to do frequent code restructuring (like in case we added an early return, or we needed assignments to go in a certain order).